layers/+spacemacs/spacemacs-defaults/keybindings.el

```diff
@@ -733,6 +734,15 @@ respond to this toggle."
   "xwc" 'spacemacs/count-words-analysis
   "xwr" 'spacemacs/randomize-words
   "x TAB" 'indent-rigidly)
+;; Version and Environment ----------------------------------------------------
+(spacemacs|spacebind
+ "Version/Environment"
+ :global
+ (("V" "Version/Environment"
```

Comment:
@syl20bnr  Extra set of `()`


@lebensterben  I think we should detect it and report as en error since it is easy to introduce when refactoring.